sdk/src/main/java/com/oursky/authgear/AuthenticateOptions.kt

```diff
@@ -15,7 +15,7 @@ data class AuthenticateOptions @JvmOverloads constructor(
      */
     var state: String? = null,
     /**
-     * Custom state.
+     * Use this parameter to provide parameters from the client application to Custom UI. The string in xState can be accessed by the Custom UI. Ignore this parameter if default AuthUI is used
```

Comment:
Please also apply this doc string update to `PromoteOptions`, `ReauthenticateOptions`

Reply:
fixed up the comment in them